data/common.yaml

```diff
@@ -1,48 +1,680 @@
+# DO NOT EDIT THIS FILE
+# Content was auto-generated with .github/update-data-common
```

Comment:
The GitHub action only replace the keys that correspond to configuration files.  If we add more keys, they will not be changed unexpectedly, but as in general hiera config in common.yml should be in `manifests/init.pp`, I worded this that way which looked more simple.